job-templates/kubernetes_containerd_1_23.json

```diff
@@ -0,0 +1,79 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "featureFlags": {
+      "enableTelemetry": true
+    },
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.23",
+      "kubernetesConfig": {
+        "useManagedIdentity": false,
+        "azureCNIURLLinux": "https://github.com/Azure/azure-container-networking/releases/download/v1.4.13/azure-vnet-cni-linux-amd64-v1.4.13.tgz",
+        "azureCNIURLWindows": "https://github.com/Azure/azure-container-networking/releases/download/v1.4.13/azure-vnet-cni-windows-amd64-v1.4.13.zip",
+        "kubeletConfig": {
+          "--feature-gates": "ExecProbeTimeout=true,KubeletPodResources=false"
+        },
+        "apiServerConfig": {
+          "--runtime-config": "extensions/v1beta1/podsecuritypolicies=true"
+        },
+        "networkPlugin": "azure",
+        "containerRuntime": "containerd",
+        "windowsContainerdURL": "https://github.com/containerd/containerd/releases/download/v1.5.8/containerd-1.5.8-windows-amd64.tar.gz"
```

Comment:
Can we target 1.6.0-beta.3?

Reply:
let's keep this for 1.23 as this is what the rest of the release branches use and update the main branch to use the latest version?  I can do that in a separate PR

Reply:
discussed offline, and since this release has hostprocess enabled we will use 1.6